sys/compat/freebsd32/syscalls.master

```diff
@@ -491,60 +500,73 @@
 256	AUE_AIO_WRITE	STD	{ int freebsd32_aio_write( \
 				    struct aiocb32 *aiocbp); }
 257	AUE_LIO_LISTIO	STD	{ int freebsd32_lio_listio(int mode, \
-				    struct aiocb32 * const *acb_list, \
+				    uint32_t *acb_list, \
```

Comment:
Is this also one of your pending makesyscalls.lua changes, or one we'll still need to make?